doozerlib/util.py

```diff
@@ -38,3 +38,8 @@ def yellow_print(msg):
 def cprint(msg):
     """Wrapper for click.echo"""
     click.echo(msg)
+
+
+def color_print(msg, color='white', nl=True):
+    """Print out a message in given color"""
+    click.secho(msg, nl=nl, bold=False, fg=color)
```

Comment:
At first this seems like a restrictive wrapper around `click.echo`. I looked above and saw you use it in the image tree printing stuff. This function makes those tree printing operations take fewer LOC or something?

Reply:
It's because `nl` and `bold` would be the same every time. It's now less LOC but it's less typing redundant stuff. If something is always the same it belongs buried in a function so there's only one place to change it ;) So yes, it's a restrictive wrapper.